diff --git a/node_modules/@rc-component/menu/es/interface.d.ts b/node_modules/@rc-component/menu/es/interface.d.ts
index 48e901b..a730c86 100644
--- a/node_modules/@rc-component/menu/es/interface.d.ts
+++ b/node_modules/@rc-component/menu/es/interface.d.ts
@@ -2,8 +2,8 @@ import type * as React from 'react';
 import type { SubMenuProps } from './SubMenu';
 interface ItemSharedProps {
     ref?: React.Ref<HTMLLIElement | null>;
-    style?: React.CSSProperties;
-    className?: string;
+    style?: React.CSSProperties | undefined;
+    className?: string | undefined;
 }
 export interface SubMenuType extends ItemSharedProps {
     type?: 'submenu';
diff --git a/node_modules/@rc-component/menu/lib/interface.d.ts b/node_modules/@rc-component/menu/lib/interface.d.ts
index 48e901b..a730c86 100644
--- a/node_modules/@rc-component/menu/lib/interface.d.ts
+++ b/node_modules/@rc-component/menu/lib/interface.d.ts
@@ -2,8 +2,8 @@ import type * as React from 'react';
 import type { SubMenuProps } from './SubMenu';
 interface ItemSharedProps {
     ref?: React.Ref<HTMLLIElement | null>;
-    style?: React.CSSProperties;
-    className?: string;
+    style?: React.CSSProperties | undefined;
+    className?: string | undefined;
 }
 export interface SubMenuType extends ItemSharedProps {
     type?: 'submenu';
