diff --git a/node_modules/@rc-component/cascader/es/Cascader.d.ts b/node_modules/@rc-component/cascader/es/Cascader.d.ts
index ba2aefb..f928137 100644
--- a/node_modules/@rc-component/cascader/es/Cascader.d.ts
+++ b/node_modules/@rc-component/cascader/es/Cascader.d.ts
@@ -55,7 +55,7 @@ interface BaseCascaderProps<OptionType extends DefaultOptionType = DefaultOption
     loadData?: (selectOptions: OptionType[]) => void;
     popupClassName?: string;
     popupMenuColumnStyle?: React.CSSProperties;
-    placement?: BaseSelectPropsWithoutPrivate['placement'];
+    placement?: NonNullable<BaseSelectPropsWithoutPrivate['placement']>;
     builtinPlacements?: BuildInPlacements;
     onPopupVisibleChange?: (open: boolean) => void;
     expandIcon?: React.ReactNode;
@@ -67,7 +67,10 @@ export interface FieldNames<OptionType extends DefaultOptionType = DefaultOption
     children?: keyof OptionType;
 }
 export type ValueType<OptionType extends DefaultOptionType = DefaultOptionType, ValueField extends keyof OptionType = keyof OptionType> = keyof OptionType extends ValueField ? unknown extends OptionType['value'] ? OptionType[ValueField] : OptionType['value'] : OptionType[ValueField];
-export type GetValueType<OptionType extends DefaultOptionType = DefaultOptionType, ValueField extends keyof OptionType = keyof OptionType, Multiple extends boolean | React.ReactNode = false> = false extends Multiple ? ValueType<Required<OptionType>, ValueField>[] : ValueType<Required<OptionType>, ValueField>[][];
+type StrictRequired<T> = {
+    [K in keyof T]-?: Exclude<T[K], undefined>;
+};
+export type GetValueType<OptionType extends DefaultOptionType = DefaultOptionType, ValueField extends keyof OptionType = keyof OptionType, Multiple extends boolean | React.ReactNode = false> = false extends Multiple ? ValueType<StrictRequired<OptionType>, ValueField>[] : ValueType<StrictRequired<OptionType>, ValueField>[][];
 export type GetOptionType<OptionType extends DefaultOptionType = DefaultOptionType, Multiple extends boolean | React.ReactNode = false> = false extends Multiple ? OptionType[] : OptionType[][];
 type SemanticName = 'input' | 'prefix' | 'suffix' | 'placeholder' | 'content' | 'item' | 'itemContent' | 'itemRemove';
 type PopupSemantic = 'list' | 'listItem';
