diff --git a/node_modules/@rc-component/qrcode/es/interface.d.ts b/node_modules/@rc-component/qrcode/es/interface.d.ts
index 62c2ac0..773938c 100644
--- a/node_modules/@rc-component/qrcode/es/interface.d.ts
+++ b/node_modules/@rc-component/qrcode/es/interface.d.ts
@@ -95,7 +95,7 @@ export type QRProps = {
     /**
      * The style to apply to the QR Code.
      */
-    style?: React.CSSProperties;
+    style?: React.CSSProperties | undefined;
     /**
      * Whether or not a margin of 4 modules should be rendered as a part of the
      * QR Code.
@@ -117,7 +117,7 @@ export type QRProps = {
     /**
      * The title to assign to the QR Code. Used for accessibility reasons.
      */
-    title?: string;
+    title?: string | undefined;
     /**
      * The minimum version used when encoding the QR Code. Valid values are 1-40
      * with higher values resulting in more complex QR Codes. The optimal
diff --git a/node_modules/@rc-component/qrcode/lib/interface.d.ts b/node_modules/@rc-component/qrcode/lib/interface.d.ts
index 62c2ac0..773938c 100644
--- a/node_modules/@rc-component/qrcode/lib/interface.d.ts
+++ b/node_modules/@rc-component/qrcode/lib/interface.d.ts
@@ -95,7 +95,7 @@ export type QRProps = {
     /**
      * The style to apply to the QR Code.
      */
-    style?: React.CSSProperties;
+    style?: React.CSSProperties | undefined;
     /**
      * Whether or not a margin of 4 modules should be rendered as a part of the
      * QR Code.
@@ -117,7 +117,7 @@ export type QRProps = {
     /**
      * The title to assign to the QR Code. Used for accessibility reasons.
      */
-    title?: string;
+    title?: string | undefined;
     /**
      * The minimum version used when encoding the QR Code. Valid values are 1-40
      * with higher values resulting in more complex QR Codes. The optimal
